### DISCUSSION TITLE
Altın Dikdörtgenler

### DISCUSSION THREAD

**Discussion by Ersan0102** [sonsuzus/sonsuzus#429] [Soru Cevap]
**Problem**
Ntane dikdörtgeniniz var. Kenarlarının oranı [1,6-1,7] arasında olan bir dikdörtgen altındır
, her ikisi de dahil. Göreviniz altın dikdörtgenlerin sayısını bulmak.

**Giriş formatı**

**İlk satır:** Tamsayı N
 dikdörtgenlerin sayısını belirten değerdir
**N'nin her biri aşağıdaki satırlar**: İki tam sayı W,H
 bir dikdörtgenin genişliğini ve yüksekliğini belirten değerlerdir.

**Çıkış biçimi:**

!!Cevabı tek satırda yazdırın!!

**Kısıtlamalar:**
1 <= N <= 10^5

1 <= G,Y <= 10^9

**Örnek Giriş:**
5
10 1
165 100
180 100
170 100
160 100

**Örnek Çıktı:**
3

**Comment by Ersan0102** [comment]
Linki: https://www.hackerearth.com/practice/algorithms/searching/linear-search/practice-problems/algorithm/almost-golden-rectangular-1c9d72c0/

**Comment by Erva-Naz** [comment]
#include <iostream>
using namespace std;
int main() {
	int n=0;
	double w=0,h=0;
	int c=0;

	cin>>n;

	for(int i=0;i<n;i++) {
		cin>>w>>h;

		if(w>h) {
			if(w/h<=1.7&& w/h>=1.6)
			c++;
		}

		else{
			if(h/w<=1.7&& h/w>=1.6)
			c++;
		}


	}

	cout << c<<endl;
}
